src/test/seratomarkerstest.cpp

```diff
@@ -0,0 +1,75 @@
+#include <gtest/gtest.h>
+#include <taglib/textidentificationframe.h>
+#include <taglib/tstring.h>
+
+#include <QDir>
+#include <QtDebug>
+
+#include "track/serato/markers.h"
+#include "util/memory.h"
+
+namespace {
+
+class SeratoMarkersTest : public testing::Test {
+  protected:
+    void parseEntry(const QByteArray inputValue, bool valid, quint32 startPosition, quint32 endPosition, QRgb color, quint8 type, bool isLocked) {
```

Comment:
Please break this line. One argument per line to trick clang.  
inputValue can be "const QByteArray&" to skip the implicit sharing code. 

Btw.: we don't use const for copy by value parameters, because it is irrelevant for the external users if the value is changed internally or not.